src/colorizer/ColorizeCanvas.ts

```diff
@@ -350,8 +355,14 @@ export default class ColorizeCanvas {
     this.pickMaterial.uniforms[name].value = value;
   }
 
+  /** Sets the current color ramp. Used when a continuous or discrete feature is selected. */
```

Comment:
just out of curiosity, why is discrete here instead of with categorical features? are discrete features usually always a ramp, ie numbers in order, just ints?

Reply:
Yes! Discrete features still use the ramps. They're essentially the same as continuous features but just fix the values at integer points.

I've been internally labeling integer features as `discrete`, and ones that are distinct categories `categorical`. I'm not sure if that breaks the convention we normally use since there's been some mixing over discrete/categorical values in conversation.